### ISSUE TITLE
Add missing citation

### ISSUE THREAD

**Issue by Benjamin-Lee** [Benjamin-Lee/deep-rules#302]
>When dealing with sequence data, holding out test data that are evolutionarily related or that share structural homology to the training data can result in overtting that is hard to detect due to the inherent relatedness of the partitioned data (cite?)

Not sure who added that but we need to fix before submitting

**Comment by agitter** [comment]
I don't know what reference the original author intended, but one possible reference is https://doi.org/10.1038/s41580-019-0176-5.  See the paragraph starting with 
> The core issue lies with data selection between the training set and the test set, which is necessary to avoid bias that would otherwise cause overfitting of the model to the training data (meaning that the program becomes unreliable when exposed to previously unseen data). The situation when this bias has not been prevented is commonly referred to as data leakage. Generally, it is assumed that random sampling is the best way to avoid selection bias. Nevertheless, this assumption does not generally hold true in biology, because biological data samples will frequently not be independent, and so random sampling is invalid. These dependencies often result either from phylogenetic effects (that is, arising from evolutionary relationships between genes or proteins) or population effects, which arise from present day experimental sampling biases.

https://doi.org/10.1093/bib/bbv082 is another option.  Or both.

**Comment by Benjamin-Lee** [comment]
I think 10.1093/bib/bbv082 is particularly useful in this context. I'll PR it. Thanks!
